configs/mask2former/mask2former_r50_lsj_8x2_50e_coco_ins.py

```diff
@@ -0,0 +1,79 @@
+_base_ = ['./mask2former_r50_lsj_8x2_50e_coco.py']
+num_things_classes = 80
+num_stuff_classes = 0
+num_classes = num_things_classes + num_stuff_classes
+model = dict(
+    panoptic_head=dict(
+        num_things_classes=num_things_classes,
+        num_stuff_classes=num_stuff_classes,
+        loss_cls=dict(class_weight=[1.0] * num_classes + [0.1])),
+    panoptic_fusion_head=dict(
+        num_things_classes=num_things_classes,
+        num_stuff_classes=num_stuff_classes),
+    test_cfg=dict(panoptic_on=False))
+
+# dataset settings
+image_size = (1024, 1024)
+img_norm_cfg = dict(
+    mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)
+pad_cfg = dict(img=(128, 128, 128), masks=0, seg=255)
+train_pipeline = [
+    dict(type='LoadImageFromFile', to_float32=True),
+    dict(type='LoadAnnotations', with_bbox=True, with_mask=True),
+    dict(type='RandomFlip', flip_ratio=0.5),
+    # large scale jittering
+    dict(
+        type='Resize',
+        img_scale=image_size,
+        ratio_range=(0.1, 2.0),
+        multiscale_mode='range',
+        keep_ratio=True),
+    dict(
+        type='RandomCrop',
+        crop_size=image_size,
+        crop_type='absolute',
+        recompute_bbox=True,
+        allow_negative_crop=True),
+    dict(type='FilterAnnotations', min_gt_bbox_wh=(1e-5, 1e-5), by_mask=True),
+    dict(type='Pad', size=image_size, pad_val=pad_cfg),
```

Comment:
Seems that pad image first than normalize does not follow the official code base https://github.com/facebookresearch/Mask2Former/blob/main/mask2former/maskformer_model.py#L194

Reply:
[FixedSizeCrop()](https://github.com/facebookresearch/detectron2/blob/f62b8a42e4d995966d0e7d7eb4049f023860b5ff/detectron2/data/transforms/augmentation_impl.py#L302-L351) is used to pad to 1024 x 1024 with pad value 128 before the images enter the forward method of maskformer_model.

While ImageList.from_tensors() is called after normalization and can pad based on size divisibility it doesn't in this case because the padded image size is always divisible when set to 1024 x 1024.

Reply:
They use pad 128 --> norm when training and norm --> pad 0 when testing.  Confirmed both cases in MaskFormer.forward() by printing the bottom right corner.  

`print(images.tensor[0, :, -1, -1])`

When training the padding is added by FixedSizeCrop() value 128 and when testing the padding is added by ImageList.from_tensors value 0 to comply with the size divisibility setting.  

Nice catch @ZwwWayne

Reply:
So in training, Pad 128 first and norm, while in testing, norm first and pad 0. I ran the test and found that the performance of [pad 128, norm] is the same as [norm, pad 0].